ThirdPartyAdapters/applovin/applovin/src/main/java/com/applovin/mediation/AppLovinNativeAdapter.java

```diff
@@ -0,0 +1,54 @@
+package com.applovin.mediation;
+
+import android.content.Context;
+import android.os.Bundle;
+import android.util.Log;
+
+import com.applovin.sdk.AppLovinSdk;
+import com.google.android.gms.ads.AdRequest;
+import com.google.android.gms.ads.mediation.MediationNativeAdapter;
+import com.google.android.gms.ads.mediation.MediationNativeListener;
+import com.google.android.gms.ads.mediation.NativeMediationAdRequest;
+
+public class AppLovinNativeAdapter
```

Comment:
Not sure if it needs to be `public`.

Reply:
This needs to be `public` so that the AdMob SDK will be able to instanciate this class using reflection.

Reply:
I'm pretty sure even if we removed `public`, Dalvik / JVM can still access the class even from a completely different package, it's the methods/fields that will be more annoying as they are scope-protected. BUT, since we mark adapters public everywhere, we should keep `public` :). Convention > Convention